src/sass/components/_box.scss

```diff
@@ -0,0 +1,90 @@
+.boxes {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.box {
+    background-color: $bg-light;
+    color: $txt-color-dark;
+    margin-bottom: $box-margin-bottom;
+    padding: $box-padding;
+    border-radius: $border-radius-lg;
+
+    &__tittle {
+        font-size: $fz-18;
+        font-weight: $fw-bold;
+        margin-bottom: $box-margin-bottom;
+        margin-top: $box-margin-top;
+    }
+    &__text {
+        line-height: $lh-more;
+        color: $txt-gray;
+    }
+
+    &--journey {
+        $journey-border-color: #e2e2e2;
```

Comment:
It could be better to declare local variables on at least a component level - for example, L8 in this file.